src/operators/rename.ts

```diff
@@ -0,0 +1,60 @@
+import { Operator, OperatorOptions, OperatorValidator } from '../operator';
+
+export interface RenameOperatorOptions extends OperatorOptions {
+  properties: { [key: string]: string };
+}
+
+export class RenameOperator implements Operator {
+  static specification = {
+    index: { required: false, type: ['number'] },
+    properties: { required: true, type: ['object'] },
+  };
+
+  readonly index: number;
+
+  readonly properties: object;
+
+  constructor(public options: RenameOperatorOptions) {
+    const { index = 0, properties = {} } = options;
+    this.index = index;
+    this.properties = properties;
+  }
+
+  handleRename(data: { [key: string]: any }) {
+    const props = Object.getOwnPropertyNames(this.properties);
+    for (let i = 0; i < props.length; i += 1) {
+      const key = props[i];
+      const val = data[key];
+      /* eslint-disable no-param-reassign */
+      delete data[key];
```

Comment:
Should we use `// eslint-disable-line no-param-reassign` rather than the eslint block comment above?  If the block is used, I believe that disables the rule for subsequent lines in the file.  I think we just make the exclusion specific to this line.